### ISSUE TITLE
TypeError: expected a character buffer object

### ISSUE THREAD

**Issue by steinnes** [bcicen/slacksocket#25]
I'm running into this repeatedly when I use mentions with a little slack bot I'm building with SlackSocket (great module btw! :-)

Not sure if I'm doing something weird, but I was easily able to "fix" it on my machine by changing the line 42 of models.py, so that `SlackEvent._get_mentions` converts each mention to `unicode` instead of `str` before calling `.translate`.

Here's the full stack trace with debugging logging enabled:

```
ERROR:root:error from callback <bound method SlackSocket._event_handler of <slacksocket.client.SlackSocket object at 0x10acc5c50>>: expected a character buffer object
  File "/Users/ses/w/pivo/venv/lib/python2.7/site-packages/websocket/_app.py", line 243, in _callback
    callback(self, *args)
  File "/Users/ses/w/pivo/venv/lib/python2.7/site-packages/slacksocket/client.py", line 367, in _event_handler
    event = SlackEvent(json_object)
  File "/Users/ses/w/pivo/venv/lib/python2.7/site-packages/slacksocket/models.py", line 31, in __init__
    self.mentions = self._get_mentions(self.event['text'])
  File "/Users/ses/w/pivo/venv/lib/python2.7/site-packages/slacksocket/models.py", line 42, in _get_mentions
    return [str(m).translate(translate_map) for m in mentions]
```

Since I'm not sure if this is somehow my fault I decided to make this issue and not a PR :-)

lib version is 0.6.3
operating system: OS X 10.11
python version:

```
$ venv/bin/python --version
Python 2.7.10
```

**Comment by bcicen** [comment]
Thanks for pointing this out @steinnes; this is in fact a py2 vs py3 bug.

I've pushed https://github.com/bcicen/slacksocket/commit/6e043ebda4b2fc229e044505703ef602ed86a105 to coerce the mentions to unicode before translating if using python2 and will include this in the next release. Thanks again for the feedback!

**Comment by steinnes** [comment]
I suspected something like that (ie. I couldn't be the only one using mentions).  Glad to help :-)

**Comment by bcicen** [comment]
fix included in v0.6.5, pushed to Pypi as well
